x-pack/plugins/ml/public/file_datavisualizer/components/experimental_badge/experimental_badge.js

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+
+import {
+  EuiBetaBadge,
+} from '@elastic/eui';
+
+export function ExperimentalBadge({ tooltipContent }) {
```

Comment:
Could you apply a `className` directly to `EuiBetaBadge` to achieve the same result here?

Reply:
the css would need to be in a shared location. so i decided it would be easier to just create a shared component instead to make it obvious that it's being overridden.